.github/workflows/node.js.yml

```diff
@@ -6,6 +6,12 @@
 #
 #   https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 #
+# This workflow requires the following GitHub Actions secrets:
+#
+#   * INFURA_API_KEY: Your API key for Infura. Obtain one at https://infura.io/
+#   * PRIVATE_KEY: The wallet private key or mnemonic for determing the accounts
+#     to use for transactions
+#
 ################################################################################
```

Comment:
We should go with node ganache module and spawn our own eth node.
Then we can also remove the dotenv file in this git repo.

Having a private key and our private infura id in public git repo is IMO not the way to go.